File: Python/Exercicios/URI/URI1828-Vitória.py
#URI Exercício 1828 por Vitória A Guardieiro

N = int(input())

for i in range(N):
	I = input().split()
	if I[0]==I[1]:
		print('Caso #%d: De novo!' %(i+1))
	else:
		if I[0]=='pedra':
			if I[1]=='papel' or I[1]=='Spock':
				print('Caso #%d: Raj trapaceou!' %(i+1))
			else:
				print('Caso #%d: Bazinga!' %(i+1))
		elif I[0]=='tesoura':
			if I[1]=='pedra' or I[1]=='Spock':
				print('Caso #%d: Raj trapaceou!' %(i+1))
			else:
				print('Caso #%d: Bazinga!' %(i+1))
		elif I[0]=='papel':
			if I[1]=='tesoura' or I[1]=='lagarto':
				print('Caso #%d: Raj trapaceou!' %(i+1))
			else:
				print('Caso #%d: Bazinga!' %(i+1))
		elif I[0]=='lagarto':
			if I[1]=='pedra' or I[1]=='tesoura':
				print('Caso #%d: Raj trapaceou!' %(i+1))
			else:
				print('Caso #%d: Bazinga!' %(i+1))
		else:
			if I[1]=='papel' or I[1]=='lagarto':
				print('Caso #%d: Raj trapaceou!' %(i+1))
			else:
				print('Caso #%d: Bazinga!' %(i+1))

File: C/URI/URI1828-Vitória.c
//URI Exercício 1828 em C por Vitória A Guardieiro

#include <stdio.h>
#include <string.h>

int jogar(char um[20], char dois[20]){
	if(strcmp(um, "tesoura")==0 && strcmp(dois, "papel")==0)	return 1;
	else if(strcmp(um, "papel")==0 && strcmp(dois, "pedra")==0)	return 1;
	else if(strcmp(um, "pedra")==0 && strcmp(dois, "lagarto")==0)	return 1;
	else if(strcmp(um, "lagarto")==0 && strcmp(dois, "Spock")==0)	return 1;
	else if(strcmp(um, "Spock")==0 && strcmp(dois, "tesoura")==0)	return 1;
	else if(strcmp(um, "tesoura")==0 && strcmp(dois, "lagarto")==0)	return 1;
	else if(strcmp(um, "lagarto")==0 && strcmp(dois, "papel")==0)	return 1;
	else if(strcmp(um, "papel")==0 && strcmp(dois, "Spock")==0)	return 1;
	else if(strcmp(um, "Spock")==0 && strcmp(dois, "pedra")==0)	return 1;
	else if(strcmp(um, "pedra")==0 && strcmp(dois, "tesoura")==0)	return 1;
	return 0;
}

main(){
	int N, a;
	char sheldon[20], raj[20];
	scanf("%d\n", &N);
	for(a=0; a<N; a++){
		scanf("%s%s", sheldon, raj);
		printf("Caso #%d: ", a+1);
		if(strcmp(sheldon, raj)==0)		puts("De novo!");
		else if(jogar(sheldon, raj)==1)	puts("Bazinga!");
		else				puts("Raj trapaceou!");
	}
}
